### PR TITLE
add custom 404 page

### PR DESCRIPTION
**Pull request by oraNod**
##### SUMMARY
Adds a custom 404 page

image

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Docs

##### AWX VERSION
n/a

##### ADDITIONAL INFORMATION
https://forum.ansible.com/t/awx-documentation-no-longer-exists-besides-contribution-guide-and-api-reference/11161

### CODE REVIEW

**Review by gundalow — COMMENTED** [review]
Thanks for doing this so quickly

Please add [cowsay 404](https://docs.ansible.com/ansible/latest/404.html)

**Comment by tvo318** [comment]
Looks great @oraNod 🎉

**Comment by oraNod** [comment]
@gundalow pushed a commit with cowsay

image

**Review by gundalow — APPROVED** [review]

**Review by tvo318 — APPROVED** [review]

File: docs/docsite/rst/404.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+.. image:: images/cow.png
+   :alt: Cowsay 404
+
+You can find different versions of the AWX documentation by opening the Read the Docs fly out menu at the bottom right.
+Here is a link to the `AWX 24.6.1 <https://ansible.readthedocs.io/projects/awx/en/24.6.1/>`_ documentation.
+
+If you have more questions or still can't find the docs you're looking for, join the `Ansible Forum <https://forum.ansible.com>`_.
+Here are some useful links for AWX users:
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `awx`, `ee`, and  `documentation`.
```
Comment (webknjaz) [inline docs/docsite/rst/404.rst:18]: Single quotes correspond to "any" role in RST, not inline code.

**Pull request merged**